src/components/home/InputForm/InputForm.tsx

```diff
@@ -11,7 +11,10 @@ import { InputField, SelectField } from '..';
 
 const schema: yup.SchemaOf<BodyInfo> = yup.object().shape({
   birth: yup.string().required(),
-  gender: yup.mixed<Gender>().oneOf(Object.values(Gender)).required(),
+  gender: yup
+    .number()
+    .oneOf(Array.from(Array(Object.keys(Gender).length / 2).keys()))
+    .required(),
```

Comment:
```ts
  gender: yup
    .mixed<Gender>()
    .test((value) => Object.values(Gender).includes(value as Gender))
    .required(),
```

Reply:
If a non `number` method (such as `mixed`) is used, then value is string(ex. "0"), resulting in a failure to determine such usage as `oneOf` or `Object.value().includes()`

Reply:
fixed, use String enums